packages/renderer/src/lib/diagnose/DiagnoseContainerEngine.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+import type { ProviderContainerConnectionInfo } from '../../../../main/src/plugin/api/provider-info';
+import DiagnoseContainerEnginePing from './DiagnoseContainerEnginePing.svelte';
+import DiagnoseContainerEngineGrabContainers from './DiagnoseContainerEngineGrabContainers.svelte';
+
+export let containerEngineRunning: ProviderContainerConnectionInfo;
+</script>
+
+<div class="flex flex-col bg-charcoal-800 border-zinc-600 m-4 p-4 text-sm">
+  <h6 aria-label="name">Name: {containerEngineRunning.name}</h6>
+  <div class="mx-4">
+    <h6 aria-label="status">Status: {containerEngineRunning.status}</h6>
+    <h6 aria-label="socket path">Socket path: {containerEngineRunning.endpoint.socketPath}</h6>
+  </div>
+  <div>Actions:</div>
+  <div class="mx-4">
+    <DiagnoseContainerEnginePing providerContainerEngine="{containerEngineRunning}" />
```

Comment:
Would be good to vertically align the responses for these 2 elements

Reply:
![image](https://github.com/containers/podman-desktop/assets/436777/3817eaf8-2045-4429-a54c-a1bdc6a56d9f)

Reply:
horizontally aligned